virt_lightning/api.py

```diff
@@ -408,16 +414,36 @@ def storage_dir(configuration, **kwargs):
     hv.init_storage_pool(configuration.storage_pool)
     return hv.get_storage_dir()
 
-
-def fetch(configuration, progress_callback=None, **kwargs):
+def custom_config_view(configuration, **kwargs):
     """
-    Retrieve a VM image from Internet.
```

Comment:
There is already a `virt_lightning.configuration` module which uses `~/.config/virt-lightning/config.ini` as configuration file. I think we can just add an extra key there pointing on the 2nd repository (or better a list of extra repositories).